templates/_layout.html

```diff
@@ -70,7 +70,8 @@
         const img = new Image()
         img.onload = function () {
           imageLoadHandler(`
-          \{\{ image\(url="${str}",
+          \{\{ image\(
```

Comment:
I think it makes more sense to have it how it was before as that is how our linter will format it